pkg/caimake/cmd/init.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/caicloud/nirvana/cli"
+	"github.com/spf13/cobra"
+)
+
+const (
+	// EnvProjectType ...
+	EnvProjectType = "PROJECT_TYPE"
+)
+
+// NewCmdInit ...
+func NewCmdInit() *cli.Command {
+	cmd := cli.NewCommand(&cobra.Command{
+		Use:   "init",
+		Short: "init a new Makefile for the project",
+		Run: func(cmd *cobra.Command, args []string) {
+			os.Setenv(EnvProjectType, cli.GetString("type"))
+			bash.Run("install.sh", args)
+		},
+	})
+	cmd.AddFlag(
+		cli.StringFlag{
+			Name:     "type",
+			EnvKey:   EnvProjectType,
+			DefValue: "go",
```

Comment:
make sure you are consistent with `go` and `golang`.  i think in someplace, you are using `go`, and someplace uses `golang`.